package.json

```diff
@@ -26,6 +26,7 @@
     "copy-webpack-plugin": "^4.6.0",
     "css-loader": "^3.0.0",
     "exports-loader": "^0.6.3",
+    "geppetto-client": "file:geppetto-client/geppetto-client",
```

Comment:
This change will make Geppetto-Application backward compatibility but at the same it will not be able to use the [refactored UI components](https://github.com/openworm/geppetto-client/tree/splitRefactoring/geppetto-ui/src). 
I think this goes in line with the changes we are proposing with the introduction of [geppetto-website](https://github.com/MetaCell/geppetto-website) but I would wait for @tarelli 👍 on this

Reply:
I didn't have success by adding an alias in webpack. I had success by removing the dependency from this package.json, and then adding the geppetto-client as a dependency of the package.json in geppetto-client https://github.com/openworm/geppetto-client/blob/feature/254/package.json#L18